.gitignore

```diff
@@ -61,5 +61,4 @@ Session.vim
 #********** IntelliJ files ******
 *.iml
 
-# built docs
-docs/
+
```

Comment:
The goal was to exclude the built docs, which should only be compiled to the `github-pages` branch - would this keep them included?

Reply:
Local builds are now excluded and built into a separate folder. There is a different build-prod make command which builds into `docs/`, which needs to stay un-ignored because travis deploys through git.